site/en/external/lockfile.md

```diff
@@ -224,3 +224,47 @@ practices:
 By following these best practices, you can effectively utilize the lockfile
 feature in Bazel, leading to more efficient, reliable, and collaborative
 software development workflows.
+
+## Merge Conflicts {:#merge-conflicts}
+
+The lockfile format is designed to minimize merge conflicts, but they can still
+happen.
+
+### Automatic Resolution {:#automatic-resolution}
+
+Bazel provides a custom
+[git merge driver](https://git-scm.com/docs/gitattributes#_defining_a_custom_merge_driver)
+to help resolve these conflicts automatically.
+
+Set up the driver by adding this line to a `.gitattributes` file in the root of
+your git repository:
+
+```gitattributes
+MODULE.bazel.lock merge=bazel-lockfile-merge
+```
+
+Then each developer who wants to use the driver has to register it once by
+following these steps:
+
+1. Install [jq](https://jqlang.github.io/jq/download/) (1.5 or higher).
+2. Run the following commands:
+
+```bash
+jq_script=$(curl https://raw.githubusercontent.com/bazelbuild/bazel/release-7.2.0/scripts/bazel-lockfile-merge.jq)
+printf '%s\n' "${jq_script}" | less # to optionally inspect the jq script
+git config merge.bazel-lockfile-merge.name   "Merge driver for MODULE.bazel.lock (Bazel 7.2.0)"
+git config merge.bazel-lockfile-merge.driver "jq -s '${jq_script}' -- %O %A %B > %A.jq_tmp && mv %A.jq_tmp %A"
```

Comment:
this places the content of the script (including comments) into my `.git/config` folder somewhere? I don't have a better suggestion.

Reply:
Yes, it ends up in the `.git/config` file as a quoted string with `\n`s.